pom.xml

```diff
@@ -126,13 +126,14 @@
                     <version>${org.codehaus.mojo.sql-maven-plugin.version}</version>
                 </plugin>
                 <plugin>
-                    <groupId>net.revelc.code</groupId>
+                    <groupId>net.revelc.code.formatter</groupId>
                     <artifactId>formatter-maven-plugin</artifactId>
                     <version>${formatter-maven-plugin.version}</version>
                     <configuration>
                         <configFile>${project.root.basedir}/eclipse/formatter.xml
                         </configFile>
                         <lineEnding>CRLF</lineEnding>
+                        <sourceDirectory>${project.basedir}</sourceDirectory>
```

Comment:
I fixed formatter-maven-plugin configuration and xml-maven-plugin configuration at  af632050d11b237e04361d2073d1a10ad755f03c commit .